pkg/api/v1/conversion.go

```diff
@@ -276,6 +276,7 @@ func Convert_v1_PodStatusResult_To_api_PodStatusResult(in *PodStatusResult, out
 		if err := json.Unmarshal([]byte(value), &values); err != nil {
 			return err
 		}
+		// TODO: Is it OK to modify in in Convert function?
```

Comment:
Sorry - I don't understand. This is v1->api conversion.

Also, shouldn't we avoid modifying "in" objects in general. This looks like anti-pattern for me.